frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -222,6 +230,8 @@ pub mod pallet {
 		NotAdminAccount,
 		// Occurs when an admin account doesn't have enough access rights to initialize a function
 		NotEnoughAccessRights,
```

Comment:
U would change to UserDoesNotHaveCorrectAccessRight

Reply:
Resolved!